Sources/YCoreUI/Extensions/UIKit/UIView+constrainCenter.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  UIView+constrainCenter.swift
+//  YCoreUI
+//
+//  Created by Dharmik Ghelani on 20/10/22.
+//  Copyright © 2022 Y Media Labs. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    /// Struct to define center attribute
+    struct Center: OptionSet {
+        let rawValue: UInt
+        static let x = Center(rawValue: 1 << 0)
+        static let y = Center(rawValue: 1 << 1)
+        static let all: Center = [.x, .y]
+    }
```

Comment:
```suggestion
    /// Center alignment options
    public struct Center: OptionSet {
        let rawValue: UInt
        /// center X
        public static let x = Center(rawValue: 1 << 0)
        /// center Y
        public static let y = Center(rawValue: 1 << 1)
        /// all (both center X and center Y)
        public static let all: Center = [.x, .y]
    }
```
This will need to be `public`
I _think_ we can hide the `rawValue`, but if not that would need to be public and need documentation comments.

Reply:
Addressed, had to make init also public, so made that with comment